hack/util.sh

```diff
@@ -51,6 +51,11 @@ capz::util::get_random_region_gpu() {
     local REGIONS=("eastus" "eastus2" "northeurope" "uksouth" "westeurope" "westus2")
     echo "${REGIONS[${RANDOM} % ${#REGIONS[@]}]}"
 }
+# all regions below must support ExtendedLocation
+capz::util::get_random_region_edgezone() {
+    local REGIONS=("eastus2euap")
```

Comment:
is this the only supported region currently?

Reply:
Yes, we have not replicated CAPI image to other edgezone region. But we will support more regions in the future.